kubeadm/scripts/PrepareNode.ps1

```diff
@@ -67,7 +67,6 @@ $env:Path += ";$global:KubernetesPath"
 
 DownloadFile $kubeletBinPath https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubelet.exe
 DownloadFile "$global:KubernetesPath\kubeadm.exe" https://dl.k8s.io/$KubernetesVersion/bin/windows/amd64/kubeadm.exe
-DownloadFile "$global:KubernetesPath\wins.exe" https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
 
 if ($ContainerRuntime -eq "Docker") {
```

Comment:
We could likely just remove all references to docker here.  It will not work with hostprocess containers and isn't support in 1.24+

Reply:
I removed all references to docker. It is only containerD now